### PR TITLE
Vr Worlds (Workshop Bounty)

### PR DESCRIPTION
**Pull request by wollygfx**
Workshop's video: https://hackclub.slack.com/files/U015HRC40H4/F01F2DD3YBC/workshop.mov

- [x] Could you check that all the assets are available? Some are missing.
- [x] Could you clean up the Markdown a bit, not the biggest deal tho.
- [x] Could you use a langagues link for repl.it, makes it easier for club members
- [x] Remember to use html throughout for markdown syntax highlighting
- [x] For the expansion demos could you add the source code
- [x] Check the grammar, just general check ups nothing big. Grammarly could probs catch them all

_Note: Github doesn't like long files (that's why it doesn't display the assets you asked me to check). Check [here](https://camo.githubusercontent.com/eeba3dffb4b6584046af66833711a33b1b78b08d0173b4fbc4fd274890316e87/68747470733a2f2f636c6f75642d646e3536787777306d2e76657263656c2e6170702f3073637265656e5f7265636f7264696e675f323032302d31312d31365f61745f362e33362e32365f706d2e676966), but it shouldn't be a problem when merging... All images will display as they would normally do._ 

@sampoder Thanks :)

### CODE REVIEW

**Review by sampoder — APPROVED** [review]
YAY! I'll trust you on the images ;) 

Re: demos, I'll leave that up to @MatthewStanciu 

Congrats!

**Comment by wollygfx** [comment]
Thanks!

**Comment by MatthewStanciu** [comment]
Some of the images still appear broken, but I'll fix it myself after merging. Great job!

**Pull request merged**
